relayer/processor/path_processor.go

```diff
@@ -14,9 +14,11 @@ import (
 )
 
 const (
-	// DurationErrorRetry determines how long to wait before retrying
+	// durationErrorRetry determines how long to wait before retrying
 	// in the case of failure to send transactions with IBC messages.
-	DurationErrorRetry = 5 * time.Second
+	durationErrorRetry = 5 * time.Second
```

Comment:
This specific timeout is for the time that the path processor will run its process again (after a send error) if it has not received updated messages from the chain processors in that time. It will most likely receive them, and every time it goes through the `processLatestMessages`, it will account for all of the new messages, so it will not retry messages that have already been acted upon by other relayers. But the actual send message retry logic will be in the next PR which I am working on now.

Reply:
Thanks for clearing that up!